packages/lumx-core/src/scss/components/list/_mixins.scss

```
@@ -44,6 +44,11 @@
 
     @include lumx-state(lumx-base-const("state", "HOVER"), lumx-base-const("emphasis", "LOW"), "dark");
 
+    &[data-focus-visible-added] {
+        outline: 2px solid lumx-color-variant("light", "N");
```

I made a mistake in the code I gave you, sorry 😬 
```suggestion
        outline: 2px solid lumx-color-variant("dark", "N");
```